### ISSUE TITLE
Doesn't start on Ubuntu 19.10

### ISSUE THREAD

**Issue by rkononov** [regolith-linux/regolith-desktop#105]
Seems like something was broken during most recent upgrades (I suspect ubuntu/gnome).
Doesn't start at all - just gray screen, nothing works (shortcuts or something).
Gnome works without issues.
I even removed completely, cleaned out all the files and reinstalled from scratch and the same.
Any ideas how to debug/which logs to check?(if I replace Exec=i3-gnome-flashback-session with just i3 it loads)

**Comment by kgilmer** [comment]
HI @rkononov , how did you install Regolith on Ubuntu 19.10?  Since I don't distribute packages for that version I'm guessing you did an upgrade from an earlier Ubuntu version.  For troubleshooting, check `/var/log/syslog` for messages from i3, gnome, or regolith during the timeframe that your login attempt occurs.

**Comment by rkononov** [comment]
Thanks I'll check syslog (yep I upgraded from the previous one)

**Comment by rkononov** [comment]
I found the only CIRITAL message in the logs around the i3 start time (below)
Is there anything I could try else - maybe build from sources?

```
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session[9550]: gnome-session-binary[9550]: WARNING: Unable to find required component 'org.gnome.SettingsDaemon.Clipboard'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session-binary[9550]: WARNING: Unable to find required component 'org.gnome.SettingsDaemon.Clipboard'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session[9550]: gnome-session-binary[9550]: WARNING: Unable to find required component 'org.gnome.SettingsDaemon.Mouse'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session-binary[9550]: WARNING: Unable to find required component 'org.gnome.SettingsDaemon.Mouse'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session[9550]: gnome-session-binary[9550]: WARNING: Unable to find required component 'gnome-flashback-init'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session-binary[9550]: WARNING: Unable to find required component 'gnome-flashback-init'
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session[9550]: gnome-session-binary[9550]: CRITICAL: We failed, but the fail whale is dead. Sorry....
Sep  1 19:42:05 norn-XPS-15-9560 gnome-session-binary[9550]: CRITICAL: We failed, but the fail whale is dead. Sorry....
```

**Comment by kgilmer** [comment]
Thanks for providing the log details @rkononov .  It may be something with `gnome-session`.  There is nothing here that provides me enough information to provide a fix or suggest more troubleshooting steps.  I plan to support 19.10 but will probably not be able to focus on this in the short-term.

**Comment by rkononov** [comment]
@kgilmer anything I could do for debugging?(any way to launch this binary in debug mode  or increase log level) - I have some development experience and really interested in fixing this, but no idea what to start with

**Comment by kgilmer** [comment]
The script that runs to launch i3 is `/usr/bin/i3-gnome-flashback`.  This gets executed upon login.  Perhaps by adding debug statements in that script when you try and start it up we can find out where in the script (assuming the problem is in there) the problem lies...

**Comment by kgilmer** [comment]
WIP: 
![regolith-eoan](https://user-images.githubusercontent.com/49683/64916008-325b8380-d729-11e9-81c6-fa86f1e96c1f.png)

**Comment by kgilmer** [comment]
This change is now available for testing in the `regolith-unstable` PPA.

**Comment by rkononov** [comment]
Thanks!
I'll test it right away (btw had a chance to debug it and it was failing on gnome-session start, unfortunately even in debug mode it was not so informative)

**Comment by rkononov** [comment]
Thanks it's fixed!
Could you point to the commit with the fix?(out of curiosity)

**Comment by kgilmer** [comment]
Yay, glad to hear it @rkononov :smile: .  Regarding the fix, I basically just installed it on a laptop and tried changing things until I could get it to work.  I found some commits in gnome-flashback related to some of the warnings I was seeing when the session was trying to start.  However it was pretty difficult to determine exactly what was wrong, and so had to resort to trial and error.  Here's eventually what I reduced the changes down to: https://github.com/regolith-linux/regolith-gnome-flashback/commit/151d7d671df79957abc521d5326dd38559c32a1f
